fe1-web/src/features/linked-organizations/screens/LinkedOrganizationsScreen.tsx

```diff
@@ -0,0 +1,432 @@
+                <Text style={Typography.paragraph}>
+                  {STRINGS.linked_organizations_enterChallengeValue}
+                </Text>
+                <Input
+                  testID="modal-input-challengeval"
+                  value={manualChallengeValue}
+                  onChange={setManualChallengeValue}
+                  placeholder={STRINGS.linked_organizations_placeholderChallengeValue}
+                />
+
+                <Text style={Typography.paragraph}>
+                  {STRINGS.linked_organizations_enterChallengeValidUntil}
+                </Text>
+                <DatePicker
+                  selected={startDate}
+                  onChange={(date: Date) => {
+                    setStartDate(date);
+                    setManualChallengeValidUntil(Timestamp.fromDate(date));
+                  }}
+                />
+
+                <PoPTextButton
+                  testID="add-manually"
+                  onPress={() => {
+                    if (
+                      !manualLaoId ||
+                      !manualPublicKey ||
+                      !manualServerAddress ||
+                      !manualChallengeValue ||
+                      manualChallengeValidUntil <= Timestamp.EpochNow()
+                    ) {
+                      toast.show(
+                        `All fields are required and Valid Until has to be in the Future`,
+                        {
+                          type: 'danger',
+                          placement: 'bottom',
+                          duration: FOUR_SECONDS,
+                        },
+                      );
+                      return;
+                    }
+
+                    const tmpOrg = new Organization({
+                      lao_id: new Hash(manualLaoId),
+                      public_key: new Hash(manualPublicKey),
+                      server_address: manualServerAddress,
+                      challenge: new Challenge({
+                        value: new Hash(manualChallengeValue),
+                        valid_until: manualChallengeValidUntil,
+                      }),
+                    });
+                    onScanData(tmpOrg.toJson());
+                    setInputModalIsVisible(!inputModalIsVisible);
+                    setManualLaoID('');
+                    setManualPublicKey('');
+                    setManualServerAddress('');
+                    setManualChallengeValue('');
+                    setManualChallengeValidUntil(Timestamp.EpochNow());
+                  }}>
+                  {STRINGS.general_add}
+                </PoPTextButton>
+              </ScrollView>
+            </Modal>
+          </ScreenWrapper>
+          <FAB
+            testID="fab-button"
+            placement="right"
+            color={accent}
+            onPress={() => {
+              setShowModal(true);
+            }}
+            icon={{ name: 'add', color: contrast }}
+            style={styles.fabStyle}
+          />
```

Comment:
Are you sure you want to use a FAB here? This is not used anywhere else in the UI. iOS usually has these actions in the top navigation bar, top right corner. Right now fe1's UI tries to stick more towards the iOS side since there is a separate Android app :)